lib/rules/no-unexpected-multiline.js

```diff
@@ -82,6 +82,10 @@ module.exports = {
 
             "BinaryExpression[operator='/'] > BinaryExpression[operator='/']"(node) {
```

Comment:
Thanks for the explanation! I almost thought the fix wouldn't be sufficient but I thought it does not hurt to open the PR to learn a bit more about eslint.